### DISCUSSION TITLE
Prompt to voice

### DISCUSSION THREAD

**Discussion by moat335a** [coqui-ai/TTS#2625] [General Q&A]
Hello. 
I am quite new to TTS and was wondering if Prompt to speech is possible with the open source models? and if yes which model should i use for this perpuse?

**Comment by erogol** [comment]
p2v is studio exclusive ATM

**Reply by vladfaust** [reply]
Any chance to open-source it due to studio sunset?

**Reply by vladfaust** [reply]
Could you at least tell the name of the model which you used for p2v? Thanks.

**Comment by Gushousekai195** [comment]
Bring.

It.

Back!

**Comment by johnbenac** [comment]
I just heard about Coqui recently, and am sad to see it go. The main draw for me is Prompt To Voice. How can I access this?
